selector.js

```diff
@@ -4,3 +4,10 @@ selector.addEventListener('click', e => {
   selector.classList.toggle('open');
   selections.classList.toggle('open');
 });
+
+document.addEventListener('click', e => {
+  if (e.target && !['header-label', 'selector', 'selections'].includes(e.target.id) && selector.classList.contains('open')) {
```

Comment:
Instead of declaring the ids inline here can we query the element above and then compare?

The point of that would be to make sure there is one source of truth for the ids in this file.

Reply:
Updated to use [`contains`](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains)!